### DISCUSSION TITLE
Volume Control doesn't work on HUAWEI MateView GT 34"

### DISCUSSION THREAD

**Discussion by delvinwidjaja** [MonitorControl/MonitorControl#1594] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Hi, first of all, thank you so much for this app.
I've been using this app with my MB Pro M1 connected to a HUAWEI MateView GT 34" (with a built-in soundbar) via Display Port.
After I've upgraded to MacOS Sonoma somehow I could only control the external monitor brightness, not the volume of the soundbar. I've taken a look at the advanced settings and it looks like this:
![Screenshot 2024-06-01 at 13 09 08](https://github.com/MonitorControl/MonitorControl/assets/44110031/158a098c-6b59-45cf-84d8-bc037584856c)

It seems the volume isn't available. Did I do something wrong?

### Expected behavior

App should be able to control the volume of the external display's soundbar

### Environment Information (please complete the following information)

```markdown
- macOS version: Sonoma 14.3.1
- Mac model: MB Pro M1
- MonitorControl version: 4.2
- Monitor(s): HUAWEI MateView GT 34" with built-in soundbar
- Monitor Cable(s)/Connection(s): DP
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
To me it seems like the app does not detect the display as suitable for DDC control. Do you have the same issue with https://github.com/waydabber/BetterDisplay on the same setup?

**Comment by delvinwidjaja** [comment]
Hi, I've tried BetterDisplay and as you said apparently, it has the same problem detecting the DDC control.
MonitorControl used to work on my setup on MacOS Ventura. Is this something that can be "fixed"?

**Reply by waydabber** [reply]
I am not sure - if it worked on Ventura, should work under newer systems as well. It might be a connection issue?
